dask_sql/physical/rex/core/literal.py

```diff
@@ -145,8 +145,13 @@ def convert(
         elif literal_type == "Null":
             literal_type = SqlTypeName.NULL
             literal_value = None
+        elif literal_type == "IntervalDayTime":
+            literal_type = SqlTypeName.INTERVAL
+            literal_value = rex.getIntervalDayTimeValue()
```

Comment:
What kind of literal_value is returned in this case? Is it a string or an integer?

Reply:
i64

Reply:
In that case we might have to update the logic in sql to python value, since the conversion there seems to assume it's a string and do some computation instead.

Reply:
I think int should be fine as long as we update the logic here as well: https://github.com/dask-contrib/dask-sql/blob/datafusion-sql-planner/dask_sql/mappings.py#L131.

I'm surprised all tests passed. Maybe none actually test the case where an interval type is passed?